src/store/index.js

```diff
@@ -141,6 +141,14 @@ export class ApplicationStore {
         // [property]: value ... filter object
     };
 
+    @observable
+    discoveryYearFilter = (node) => {
```

Comment:
this isn't an observable. Observables don't depend on other values that would be a computed value. But even this isn't a computed value. A computed value is a property getter than depends on the value of other observables or computed values. 

you could: 
 1. restructure it similar to the `filterFunc` so a function generator
 1. make it a regular function that also gets the `year` as argument and store the `year` when generating the `filterFunc`

Reply:
Ah, I thought it would be some type of reactive value based on the fact that it depends on `this.year`. But I think you're saying that in order for it to depend that way on the year, I would need to make it a function generator, so that a new function is created whenever `this.year` changes, right?

I will take a look at using strategy (1) from your suggestions.

Reply:
I tried your first suggestion--let me know if it looks right.